### DISCUSSION TITLE
候选框透明

### DISCUSSION THREAD

**Discussion by 971230** [rime/home#1488] [General]
1️⃣小狼毫候选框可不可以设置半透明这种；
2️⃣候选框目前能不能添加背景图片这种，以实现win11的这种效果：
<img width="398" alt="PixPin_2024-03-08_16-53-24" src="https://github.com/rime/home/assets/37500200/88392ed9-f654-4c20-8f16-80e8349d7d1e">

**Comment by fxliang** [comment]
图片背景平铺试过，但是有些系统应用中无法绘制图片，算了

![}WZLRES1F@B@U6SOT1G9U5U](https://github.com/rime/home/assets/4023160/089217e2-357f-4dbc-9437-6e2dd3dbd7f1)

不正常的时候
![H(TYV30QW8JQ EB 4KU85~T](https://github.com/rime/home/assets/4023160/ecda5113-210b-4fa2-b2cb-6b6b0c920fb6)

**Reply by 971230** [reply]
这个是有技术壁垒，原因暂时不知道是么？还是咱们软件架构当时没有考虑到
